src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -0,0 +1,41 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.dto.UserDto;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/user")
+@CrossOrigin("*")
+public class UserController {
+
+    @GetMapping
+    public ResponseEntity<List<UserDto>> getUserList() {
+        List<UserDto> userList = new ArrayList<>();
+        userList.add(new UserDto(1L,"user1",1,15234));
+        userList.add(new UserDto(2L,"user2",0,45678));
+
+        return ResponseEntity.ok(userList);
+    }
+    @GetMapping(value = "{userId}")
```

Comment:
I think (value = "/{userId}") or only @GetMapping("/{id}") should be better. But your version is OK. Spring is very intelligent and will add / if nessesery.

In the documentation we don't have a get user by id endpoint requirement, it's very strange. I wonder if such an endpoint is necessary or just such a method to extract from repository. What do You think?